dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/TemplateLayoutRow.java

```diff
@@ -49,31 +48,22 @@ public void setValue ( String value ) {
 
         //Now based on this value (YUI layout class) we will set the row layout type
         if ( value.equals( YUI_ONE_COLUMN_CLASS ) ) {//1 Column (100)
```

Comment:
it looks like something you can handle by map, key value
you will get Log(N) instead of O(N)